app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -202,7 +202,10 @@ private class BrowserIWebViewCallback(
 
     private var fullscreenCallback: IWebView.FullscreenCallback? = null
 
-    override fun onPageStarted(url: String) {}
+    override fun onPageStarted(url: String) {
+        browserFragment.browserOverlay.updateNavigationButtons()
```

Comment:
In focus, sebastian intended to remove these callbacks and use SessionObservers instead – I'm not sure this is desirable #415 but that's the way the code is now. So this translates to:
```kotlin
session.loading.observe(... {
    // do the thing when true
```

I think we already have such an observer in this file.